src/components/filterByDuration/FilterByDuration.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputFormField from './InputFormField';
+import ButtonFormField from './ButtonFormField';
+import './FilterByDuration.css';
+
+class FilterByDuration extends React.Component {
+  handleSubmit = (event) => {
+    event.preventDefault();
+    // const data = JSON.stringify(this.state);
+    return this.state;
+  };
+
+  render() {
+    const { startTime, endTime, handleChange } = this.props;
+    return (
+      <form className="container" onSubmit={this.handleSubmit}>
+        <InputFormField
+          name="startTime"
+          value={startTime}
+          onChange={handleChange}
+        >
+          De :
+        </InputFormField>
+        <InputFormField name="endTime" value={endTime} onChange={handleChange}>
+          à :
+        </InputFormField>
+
+        <ButtonFormField onClick={this.handleSubmit} />
+      </form>
+    );
```

Comment:
Some users won't have time restaint, so we should add a button "Peu importe !" or "J'ai tout mon temps !" or something like that.

Reply:
Ok for the next sprint